client/printing/print_client.js

```diff
@@ -2,19 +2,18 @@ const grpc = require('grpc');
 const fs = require('fs')
 var messages = require('./print_pb');
 var services = require('./print_grpc_pb');
-const printOptions = {
-    'sides': 'one-sided',
-    'page-ranges': 'NA'
-};
 
-function main() {
-    let contents = fs.readFileSync(__dirname + '/../sample.pdf', 'base64');
+function main(raw, copies, sides, pageRanges, destination) {
```

Comment:
can you name this something other than main?
```suggestion
function sendPrintRequest(raw, copies, sides, pageRanges, destination) {
```